rpc_proxy.go

```diff
@@ -200,7 +263,29 @@ func (p *rpcProxy) director(ctx context.Context,
 
 	outCtx := metadata.NewOutgoingContext(ctx, mdCopy)
 
-	return outCtx, p.lndConn, nil
+	// Direct the call to the correct backend. For lnd we _always_ have a
+	// client connection, no matter if it's running in integrated or remote
+	// mode. For all other daemons the request shouldn't get here in
+	// integrated mode (after all, the director only picks up calls that the
+	// gRPC server itself would throw a 404 for) so we throw an error
+	// message for them if they're not in remote mode.
+	switch {
+	case isLndURI(requestURI):
+		return outCtx, p.lndConn, nil
+
+	case isFaradayURI(requestURI) && p.cfg.FaradayMode == ModeRemote:
+		return outCtx, p.loopConn, nil
```

Comment:
Should this be `p.faradayConn`?

Reply:
Actually meant to select line #280 below.
```go
	case isFaradayURI(requestURI) && p.cfg.FaradayMode == ModeRemote:
		return outCtx, p.loopConn, nil
```

Reply:
Most definitely. Great catch, thank you!